code-for-canada/project-thundercat
# [Inbox] Inbox Columns Structure

# Description

Creation of stubbed "Inbox" structure on inbox page with minimal functionality.

Later PRs will fix:

- Details to preview
  - Styling of preview
  - Styling of sidebar
  - Replacing 'size-nav' ids/classes
  - Accessibility of sidebar
- Details of Email
  - Styling of Email
- State in Redux
- Move Inbox.jsx into Commons

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

![image](https://user-images.githubusercontent.com/2746350/55023583-7f109e00-4fd3-11e9-8c74-491b0d48b05a.png)
![image](https://user-images.githubusercontent.com/2746350/55023602-8e8fe700-4fd3-11e9-9c94-90208086ce7f.png)


# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Protype -> Start eMIB
2. Navigate through pre-test instuctions
3. Start Test
4. Select Inbox
5. Verify that a scrollable sidebar appears with some email data.
5. Clicking on the "buttons" in the sidebar changes the displayed stubbed email.

# Checklist

Applicable for all code changes.
